Source/reactions/Castro_react.cpp

```diff
@@ -387,3 +395,123 @@ Castro::react_state(Real time, Real dt)
 }
 
 #endif
+
+
+
+bool
+Castro::valid_zones_to_burn(MultiFab& State)
+{
+
+    // The default values of the limiters are 0 and 1.e200, respectively.
+
+    Real small = 1.e-10;
+    Real large = 1.e199;
+
```

Comment:
this seems a little fragile

Reply:
Yeah, I can do better than this if we record what the default values in _cpp_parameters are.